New playbooks for xdr

mayagoldb opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46400
fixes: https://github.com/demisto/etc/issues/46399
fixes: https://github.com/demisto/etc/issues/46335


## Description
The following playbooks are included:
- Handle False Positive Alerts
- Enrichment for Verdict

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

altmannyarden commented:
@jrosenpanw Can you please review at least the playbook description?

altmannyarden commented:
@julieschwartz18 Hi, its like the other PR you review (https://github.com/demisto/content/pull/18000)
Can you review this one as well?

altmannyarden commented on Packs/CommonPlaybooks/Playbooks/playbook-Enrichment_for_Verdict_README.md:
```diff
@@ -0,0 +1,70 @@
+This playbook performs enrichment on different IOC types, and returns the information needed to establish the alert's verdict.
```
Also checks the closing reason of previous alerts :)

altmannyarden commented on Packs/CommonPlaybooks/Playbooks/playbook-Handle_False_Positive_Alerts_README.md:
```diff
@@ -0,0 +1,38 @@
+This playbook handles false positive alerts.
```
This playbook handles false positive alerts.
The playbook will create an alert exclusion, exception or add a file to allowlist based on the alert fields and playbook inputs.

julieschwartz18 commented:
Doc review complete for the playbook-WildFire_Malware.yml file. @mayagoldb - please confirm the following:
- line 790 - the description for the close alert task is "commands.local.cmd.close.inv" - is this ok?
- aligned the XSIAM playbook descriptions with the files that were previously updated in [Xsiam sub playbooks https://github.com/demisto/content/pull/18000]:(https://github.com/demisto/content/pull/18000)

julieschwartz18 commented:
@mayagoldb - please regenerate the README files since descriptions were updated throughout the yaml files.

mayagoldb commented:
@julieschwartz18 
**WildFire Malware:**
- line 790 - I updated the description.

**playbook-Handle_False_Positive_Alerts.yml:** 
- Updated all the descriptions to be more elaborate.

**Enrichment_for_Verdict:**
- updated all the descriptions for suspicious keys to: 'Set the SuspectedVerdict key in context to Suspicious Domain.' (for example).
- Updated the dbot score context key description.

altmannyarden commented on Packs/Core/Playbooks/playbook-WildFire_Malware_README.md:
```diff
@@ -0,0 +1,85 @@
+In this phase, the playbook  executes the **Containment Plan** playbook with the file hash as an input.  
+Phase 2:  
+When the playbook continues, if other suspicious IOCs are found in the investigation, the next phase of response is executed, which includes the **Containment Plan** playbook with the found IOCs as inputs.
+
```
For each phase - let's explain which of the containment actions are by default true to run automatically
